src/cli/main.rs

```diff
@@ -22,6 +22,7 @@ mod download_tracker;
 mod errors;
 mod help;
 mod job;
+mod markdown;
```

Comment:
Its a little awkward because the types were comingled; so we can't not change the md stuff. TBH I think having it in the one commit isn't going to make bisection a problem - and thats all I'd really worry about history wise.

Reply:
Okay, yeah, looking through again I agree it'd be a pain.